src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ParameterAssignmentCheck.java

```diff
@@ -50,6 +50,28 @@
  * &lt;module name=&quot;ParameterAssignment&quot;/&gt;
  * </pre>
  * <p>
+ * Example :
+ * </p>
+ * <pre>
+ * class MyClass {
+ *   int method1(int parameter1, int parameter2) {
+ *     parameter2 = 2;   // violation
+ *     parameter1 -= 2;  // violation
+ *     return parameter1;
+ *   }
+ * }
+ *
+ * class MyClass {
+ *   int method2(int parameter1, int parameter2) {
+ *     int local = parameter1;
+ *     if (parameter2 > 50) {
```

Comment:
@ek08 ,
please review CI failures yourself to resolve them
please fix:
https://travis-ci.org/github/checkstyle/checkstyle/jobs/745456605#L3347

Reply:
done